src/legacy/ui/public/styles/_legacy/components/_sidebar.scss

```diff
@@ -52,40 +52,43 @@
       }
     }
 
-      .sidebar-item-title,
-      .sidebar-item-text {
-        margin: 0;
-        padding: $euiSizeXS $euiSizeS;
-        text-align: center;
-        width: 100%;
-        border: none;
-        border-radius: 0;
-      }
+    .sidebar-item-title,
```

Comment:
Out of scope for this PR, but we should move this css to our regular class name style at some point.